client/index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE HTML>
+<html lang="en">
+
```

Comment:
I think we all need to add a linter to our code (and all of us should use the same one) - so that our code is consistent in style and has been combed over for syntax errors for us. In the G learn they suggest we use AirBnB's style guide as our linter. I need to add this to my code as well. We could choose any linter we want - AirBnB's is just Hack Reactor's suggestion. I think this would have helped catch the typo at the bottom of database/seed.js

Reply:
Good idea Shain, I just setup Eslint and configured it with the Air Bnb style guide.

Reply:
Agreed! Added eslint.